uw_sws/models.py

```diff
@@ -235,6 +235,53 @@ def __eq__(self, other):
                 type(self) == type(other) and
                 self.__key() == other.__key())
 
+    def __lt__(self, other):
+        if self.year < other.year:
+            return True
+        elif self.year > other.year:
+            return False
+
+        return (Term._quarter_to_int(self.quarter) <
+                Term._quarter_to_int(other.quarter))
+
+    def __le__(self, other):
+        if self.year < other.year:
+            return True
+        elif self.year > other.year:
+            return False
+
+        return (Term._quarter_to_int(self.quarter) <=
+                Term._quarter_to_int(other.quarter))
+
+    def __gt__(self, other):
+        if self.year > other.year:
+            return True
+        elif self.year < other.year:
+            return False
+
+        return (Term._quarter_to_int(self.quarter) >
+                Term._quarter_to_int(other.quarter))
+
+    def __ge__(self, other):
+        if self.year > other.year:
+            return True
+        elif self.year < other.year:
+            return False
+
+        return (Term._quarter_to_int(self.quarter) >=
+                Term._quarter_to_int(other.quarter))
+
+    @staticmethod
+    def _quarter_to_int(quarter):
+        if quarter == 'winter' or quarter == 'Winter':
```

Comment:
prefer something like: quarter.lower() == Term.WINTER
We want to use the same integer values defined in SDB:
1=winter
2=spring
3=summer
4=autumn